code/modules/mob/living/carbon/human/human.dm

```diff
@@ -706,7 +706,7 @@
 
 /mob/living/carbon/human/singularity_pull(S, current_size)
 	if(current_size >= STAGE_THREE)
-		var/list/handlist = list(l_hand, r_hand)
+	var/list/handlist = list(l_hand, r_hand)

```

Comment:
I intentionally did it to see if I can't get rid of whatever first triggered that error.